config/routes.rb

```diff
@@ -1,9 +1,11 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-  
-  devise_for :users
-  # Defines the root path route ("/")
-  root "users#index"
+
+  # Web routes
+
+  devise_for :users # Routes for authentication
```

Comment:
- [ ] It would be better if we are able to test our authentication routes ie. `sign_in` and `sign_up`. However, when I enter the command `rails routes` here is what I got as seen in the screenshot below
![Screenshot from 2022-11-11 02-01-36](https://user-images.githubusercontent.com/46388113/201246188-589a5c84-9b64-4994-8a8c-2baf1e7834c1.png)

Let us focus on these routes; `POST sign_in` and `GET sign_up` as seen in the routes. The following screenshot show how I tested these endpoints in POSTMAN

Making the request to `GET sign_up` I got `200` status code
![Screenshot from 2022-11-11 02-06-07](https://user-images.githubusercontent.com/46388113/201246808-936aec50-e35b-46c1-8cd9-489337189662.png)

Since I got the `200` code it is presumed that the request was successful, now I'm going to make a POST request to `POST users/sign_in` with the same credentials as seen below, it returns `422` unprocessable entity

![Screenshot from 2022-11-11 02-08-57](https://user-images.githubusercontent.com/46388113/201247084-afdc8f5e-31f8-431e-897a-814ff635c1aa.png)

I also made a POST request to `users/sign_in` although it was not in the routes as a post request, here was the result as seen below. In this instance I got a `404` not found
![Screenshot from 2022-11-11 02-11-50](https://user-images.githubusercontent.com/46388113/201247505-09f842fa-8e1f-4791-80ca-8ac4fe5cdbc9.png)

I'm unable to successfully test the authentication routes, kindly investigate this error and fix it accordingly

Reply:
Hello @addod19 ,
Thanks for including all these graphic examples, but I am not familiar with Postman, so I don't get you very well. Could you explain to me again the problem with the routes users/sign_in?
Thanks.

Reply:
Hello @addod19 ,
Another detail that I don't get is that you said I don't have users/sing_in as a POST request in the routes, but I see it in the routes
![image](https://user-images.githubusercontent.com/53547956/201255581-dcf3e4a9-04f3-4b5d-b74a-b076aa6c7e28.png)
Could you explain me?

Reply:
@Yes I can. What I mean is that I am unable to sign in the application. As code reviewers we are to test the routes. I have tested it on my system and here is the result. Sorry for the misunderstanding